### ISSUE TITLE
tts_speech = torch.cat(tts_speechs, dim=-1).cpu() - why cpu tho ?

### ISSUE THREAD

**Issue by darkacorn** [THUDM/GLM-4-Voice#64]
tts_speech = torch.cat(tts_speechs, dim=-1).cpu()

https://github.com/THUDM/GLM-4-Voice/blob/main/flow_inference.py

uses everything on cpu ? that would be slower then on gpu i assume ?
